### DISCUSSION TITLE
[Unstable] Submarine devices not deteriorating

### DISCUSSION THREAD

**Discussion by Vykn** [FakeFishGames/Barotrauma#14117] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I completed a multiplayer campaign and all throughout I did not notice any devices breaking. Assistants weren't completing Starter Quest because junction boxes weren't breaking, and the mechanic that I hired at the start of the game only repaired a total of 60 mechanical devices across an entire 18 hour campaign while being the only npc assigned to repair those devices. (And a lot of those were likely EMPs in the ending sequence)

During this campaign I did tend to travel very quickly through the levels, (often spending only 3-10 minutes in each), but after completing the campaign I enabled cheats and had a look at the devices, and they either weren't degrading even while the deterioration timer ran out, or the deterioration timer did not tick down at all.

The save:
[Ironman Gaming.zip](https://github.com/user-attachments/files/15788845/Ironman.Gaming.zip)

### Reproduction steps

1. Load the save
2. undock, and wait around, run the devices, and observe that they are not deteriorating, and some devices do not tick down the deterioration timer

### Bug prevalence

Just once

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.5.4.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fix coming up in the next unstable build, closing
